cobc/typeck.c

```diff
@@ -14389,6 +14401,8 @@ cb_emit_string (cb_tree items, cb_tree into, cb_tree pointer)
 	cb_tree l;
 	cb_tree end;
 	cb_tree dlm;
+	cb_tree st = cb_build_direct ("string_st", 0);
+	cb_tree pst = cb_build_direct ("&string_st", 0);
```

Comment:
I tried that:
```c
cb_tree st = cb_build_direct ("string_st", 0);
...
CB_BUILD_CAST_ADDRESS (st);
```
(same with `CB_BUILD_CAST_ADDR_OF_ADDR`)

but I end up with a
```
cobc: unexpected tree tag: DIRECT
cobc: codegen.c:1292: internal compiler error
```
error. I assume I should not have used `cb_build_direct` in `cb_tree st = cb_build_direct ("string_st", 0);`. I currently don't see how I'm supposed to build the `string_st` identifier (structure pointer) otherwise. Moreover, it doesn't look like there other similar examples with known identifiers used as arguments. I can dig more but if you have that information, I'm interested.

Reply:
@GitMensch Do you have an answer for this remark? I mistakenly marked it as resolved so you probably did not notice it.